clang/lib/CodeGen/CodeGenFunction.cpp

```diff
@@ -2755,9 +2757,18 @@ CodeGenFunction::CGCapturedStmtInfo::~CGCapturedStmtInfo() { }
 CodeGenFunction::SanitizerScope::SanitizerScope(CodeGenFunction *CGF)
     : CGF(CGF) {
   assert(!CGF->IsSanitizerScope);
+  assert(!ApplyTrapDI);
   CGF->IsSanitizerScope = true;
 }
 
+CodeGenFunction::SanitizerScope::SanitizerScope(
+    CodeGenFunction *CGF, ArrayRef<SanitizerKind::SanitizerOrdinal> Ordinals,
+    SanitizerHandler Handler)
+    : SanitizerScope(CGF) {
```

Comment:
```suggestion
    : SanitizerScope(CGF),  ApplyTrapDI(std::make_unique<ApplyDebugLocation>(
      *CGF, CGF->SanitizerAnnotateDebugInfo(Ordinals, Handler))) {
```

Reply:
"error: an initializer for a delegating constructor must appear alone"